### PR TITLE
Support cronjobs

### PR DESCRIPTION
**Pull request by pierDipi**
SinkBinding tests use CronJobs, this will replace the `job` package in eventing

# Changes

- Support cronjobs

### CODE REVIEW

File: pkg/resources/cronjob/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    {{ end }}
+  {{ end }}
+  {{ if .labels }}
+  labels:
+    {{ range $key, $value := .labels }}
+    {{ $key }}: {{ $value }}
+    {{ end }}
+  {{ end }}
+spec:
+  schedule: "* * * * *"
```
Comment (creydr) [inline pkg/resources/cronjob/cronjob.yaml:19]: Could we parameterize the schedule too?

Comment (pierDipi) [reply pkg/resources/cronjob/cronjob.yaml:19]: Once we need it?

Comment (creydr) [reply pkg/resources/cronjob/cronjob.yaml:19]: Ok. Can do this as a follow up when we need it.
/lgtm

**Pull request merged**